openapi/api_session.template.yaml

```diff
@@ -134,6 +167,10 @@ definitions:
     $ref: "../node_modules/@pagopa/io-functions-commons/openapi/definitions.yaml#/FiscalCode"
   ProblemJson:
     $ref: "../node_modules/@pagopa/io-functions-commons/openapi/definitions.yaml#/ProblemJson"
+  UnlockCode:
+    type: string
+    minLength: 9
+    maxLength: 9
```

Comment:
```suggestion
  UnlockCode:
    type: string
    pattern: ^\d{9}$
```

Reply:
Done in [948ec83](https://github.com/pagopa/io-backend/pull/1056/commits/948ec83b530bdea78ebd362460a27d052cb2bedc)
I had to fix the pattern using a double backslash otherwise the generated type wouldn't have decoded correctly the unlock code.